scripts/run_celery_beat.sh

```diff
@@ -2,4 +2,5 @@
 
 set -e
 
+celery -A run_celery.notify_celery purge -f || true
```

Comment:
The `|| true` ensures that if the purge fails the beat on the next line will still start.

Reply:
I think we should be careful with adding purge to a production script. It seems this command will delete all pending tasks. My question would be, would that delete only tasks that we consider invalid, or valid ones too? How do we differentiate these?

Reply:
oh very good catch, we definitely don't want to purge in prod.